pybtp/btp/gap.py

```diff
@@ -594,46 +632,39 @@ def gap_passkey_entry_rsp(bd_addr, bd_addr_type, passkey):
     gap_command_rsp_succ()
 
 
-def gap_reset():
-    logging.debug("%s", gap_reset.__name__)
-
+def gap_passkey_confirm_rsp(bd_addr, bd_addr_type, passkey):
+    logging.debug("%s %r %r", gap_passkey_confirm_rsp.__name__, bd_addr,
+                  bd_addr_type)
     iutctl = get_iut()
-    iutctl.btp_socket.send(*GAP['reset'])
 
-    gap_command_rsp_succ()
+    data_ba = bytearray()
+    bd_addr_ba = addr2btp_ba(bd_addr)
 
+    data_ba.extend(chr(bd_addr_type).encode('utf-8'))
+    data_ba.extend(bd_addr_ba)
 
-def gap_passkey_entry_req_ev(bd_addr=None, bd_addr_type=None):
-    logging.debug("%s %r %r", gap_passkey_entry_req_ev.__name__, bd_addr,
-                  bd_addr_type)
-    iutctl = get_iut()
+    if isinstance(passkey, str):
+        passkey = int(passkey, 32)
```

Comment:
As this command has almost the same structure as `gap_passkey_entry_rsp` i think it's just a copy-paste error - but this begs the question why there is 32 as a base :) I think in both places it can be default (without any base, so base = 10) because we are just making sure all passkeys have the same type.